health_check/health_serializer.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class HealthStatusSerializer(serializers.Serializer):
+    name = serializers.CharField(source='identifier', default=None)
+    status = serializers.CharField(max_length=20, source='pretty_status', default=None)
```

Comment:
I think the status can be longer than 20 characters. For example the status of the Celery ping check can include the names of the Celery queues and those alone can exceed this limit.

Reply:
Okay, I left it unbound